main.go

```diff
@@ -1242,7 +1244,7 @@ func sidekickMain(ctx *cli.Context) {
 			})
 		default:
 			console.Infof("caught signal '%s'\n", signal)
-			os.Exit(1)
+			os.Exit(globalExitCode)
```

Comment:
We are not so keen to add new flags. I think all what you need to do is to run os.Exit(143) when we receive syscall.SIGTERM. Kubernetes will understand that and will not flag an error for the container